cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -484,7 +490,62 @@ def test_custom_button_dialog_service_archived():
     Bugzilla:
         1439883
     """
-    pass
+    service, vm = service_vm
+    group, obj_type = button_group
+
+    with appliance.context.use(ViaUI):
+        button = group.buttons.create(
+            text=fauxfactory.gen_alphanumeric(start="btn_"),
+            hover=fauxfactory.gen_alphanumeric(start="hover_"),
+            dialog=dialog,
+            system="Request",
+            request="InspectMe",
+        )
+    request.addfinalizer(button.delete_if_exists)
+
+    for with_vm in [True, False]:  # [vm, archive_vm]
+        if not with_vm:
+            # Make vm archive by deleting vm from provider side
+            vm.mgmt.delete()
+            vm.wait_for_vm_state_change(
+                desired_state="archived", timeout=720, from_details=False, from_any_provider=True
+            )
+
+        for context in [ViaUI, ViaSSUI]:  # check execution with UI and SSUI
```

Comment:
Same question about parametrization here.

Reply:
I think we discuss this before also... Trying to avoid create/delete(form provider side) `VM` operations as we just need to execute button on SSUI with same setup.